libraries/AP_SmartAudio/AP_SmartAudio.cpp

```diff
@@ -60,10 +60,7 @@ bool AP_SmartAudio::init()
         _port->configure_parity(0);
         _port->set_stop_bits(2);
         _port->set_flow_control(AP_HAL::UARTDriver::FLOW_CONTROL_DISABLE);
-        //_port->set_unbuffered_writes(true);
-        _port->set_options((_port->get_options()
-            | AP_HAL::UARTDriver::OPTION_NODMA_TX | AP_HAL::UARTDriver::OPTION_NODMA_RX)
-            & ~AP_HAL::UARTDriver::OPTION_RXINV);
+        _port->set_options(_port->get_options() & ~AP_HAL::UARTDriver::OPTION_RXINV);
```

Comment:
When config the uart as half-duplex, is not the same ?

Reply:
Yeah, I'm not wedded to this - this is what RCProtocol does. I was experimenting with UART options because of connection issues.